ui/app/AppLayouts/Profile/views/ContactsView.qml

```diff
@@ -279,7 +275,6 @@ SettingsContentBase {
             //                        clip: true
```

Comment:
I think we can remove all those commented lines now. We didn't implement them in years, so we'll probably just do it from scratch anyway if we get there